templates/prompt-for-mfa-backupcode.php

```diff
@@ -13,7 +13,7 @@
 }
 
 ?>
-<form action="<?= htmlentities($this->data['formTarget']); ?>">
+<form action="<?= htmlentities($this->data['formTarget']); ?>" method="POST">
```

Comment:
Did you verify that doing this as a `POST` works? I tried that at some point for one of these forms and hit some problem that convinced me to just keep using `GET`. Unfortunately, I don't remember what it was at the moment.

Reply:
Yep, it is all working